README.md

```diff
@@ -116,12 +118,12 @@ Storing machine: please wait
 
 ### Running in production mode
 
-Once you built the Cartesi Machine snapshot, you can run the application with Sunodo by running `make run`.
+Once you built the Cartesi Machine snapshot, you can run the application with cartesi by running `make run`.
 After some time, you should see the output below in your terminal.
 You can now send advance-state and inspect-state inputs to the Rollmelette application inside the Cartesi machine.
 
 ```
-8db686eb-validator-1  | host-runner: ENABLE_SUNODO_NODE is not set, skip the host mode...
+8db686eb-validator-1  | host-runner: ENABLE_cartesi_NODE is not set, skip the host mode...
```

Comment:
plz verify the case

Reply:
@fabiooshiro Solved here: https://github.com/rollmelette/rollmelette/pull/7/commits/7a9cdd83f88fe45a1dedbeea4ed05247353526d9